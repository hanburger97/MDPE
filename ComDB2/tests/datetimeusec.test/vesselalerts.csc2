// vesselalerts.csc2
// Schema for table, "vesselalerts"

// This is the "ondisk" tag; it defines the data actually stored on disk
schema
{
  cstring  alert_id[40]                    // alert ID, synthetic key, most likey python UUID
  int      uuid                            // owner of the alert
  vutf8    name[100]                       // name of the alert
  vutf8    note[100]                       // custom note associated with the al
  short    active                          // Boolean which determines if an ale is active. 1 = Active 0 = Inactive
  datetime created                         // Timestamp of when the alert was fi inserted into this table
  datetime modified                        // Timestamp of when this record was  modified
  datetime last_sent             null=yes  // Timestamp of when this alert was l sent
  vutf8    delivery_options[100]           // Alert delivery options (JSON strin)
  vutf8    imo_criteria[1000]              // The criteria for determinig which  this alert applies to (JSON string)
  vutf8    alert_action[1000]              // Actions which the ship must take i order to trigger an alert (JSON string)
}

keys
{
      "ALERT_ID" = alert_id
  dup "UUID" = uuid
}
