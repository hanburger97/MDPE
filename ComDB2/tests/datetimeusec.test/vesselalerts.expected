(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows updated=1)
